django/core/management/commands/makemigrations.py

```diff
@@ -505,9 +507,9 @@ def all_items_equal(seq):
                         if self.scriptable:
                             self.stdout.write(writer.path)
                 elif self.verbosity == 3:
-                    # Alternatively, makemigrations --merge --dry-run --verbosity 3
-                    # will log the merge migrations rather than saving the file
-                    # to the disk.
+                    # Alternatively, makemigrations --merge --dry-run
+                    # --verbosity 3 will log the merge migrations rather than
+                    # saving the file to the disk.
```

Comment:
In general, I allowed code (and psuedo-code and shell commands) that was inlined in a paragraph to be wrapped as ordinary text. (There were several cases that already did that, and the autofixer merely changed the wrap point.)